indico/modules/events/abstracts/__init__.py

```diff
@@ -108,3 +108,15 @@ def _get_notification_placeholders(sender, **kwargs):
         obj = getattr(placeholders, name)
         if issubclass(obj, Placeholder):
             yield obj
+
+
+@signals.event.sidemenu.connect
+def _extend_event_menu(sender, **kwargs):
+    from indico.modules.events.layout.util import MenuEntryData
+
+    return MenuEntryData(
```

Comment:
We generally use alignment with opening delimiter.
https://www.python.org/dev/peps/pep-0008/#indentation